lib/ansiblelint/__main__.py

```diff
@@ -24,6 +24,7 @@
 import logging
 import pathlib
 import sys
+from argparse import Namespace
 from typing import Any, Set
```

Comment:
The typical mypy usage idiom is
```suggestion
from typing import Any, Set, TYPE_CHECKING

if TYPE_CHECKING:
    from argparse import Namespace
```